src/DotNetWorker.Core/Http/HttpResponseDataExtensions.cs

```diff
@@ -100,9 +117,35 @@ public static ValueTask WriteAsJsonAsync<T>(this HttpResponseData response, T in
             ObjectSerializer serializer = response.FunctionContext.InstanceServices.GetService<IOptions<WorkerOptions>>()?.Value?.Serializer
                  ?? throw new InvalidOperationException("A serializer is not configured for the worker.");
 
-            return WriteAsJsonAsync(response, instance, serializer, contentType, cancellationToken);
+            return WriteAsJsonAsync(response, instance, serializer, contentType, HttpStatusCode.OK, cancellationToken);
+        }
+      
+        /// <summary>
+        /// Write the specified value as JSON to the response body using the default <see cref="ObjectSerializer"/> configured for this worker.
+        /// The response content-type will be set to the provided <paramref name="contentType"/> and the status code set to 200.
```

Comment:
nit: Looks like this needs to be updated, as this will set to the status provided as the parameter.